### PR TITLE
Feature/full edit event form

### PR DESCRIPTION
**Pull request by chooidos**
add map, datepicker, duration field

### CODE REVIEW

File: client/src/events/components/EditEventForm.js [hunk 1]
```diff
@@ -238,6 +270,44 @@ const EditEventForm = props => {
         </div>
       </div>
```
Comment (yushevchuk) [inline client/src/events/components/EditEventForm.js]: Remove space, please.

File: client/src/events/components/EventItem.js [hunk 2]
```diff
@@ -65,38 +59,34 @@ const EventItem = props => {
...
+            <div className="">
               <h3>Details</h3>
               <p>{props.event.description}</p>
             </div>
           </div>
-          <div className='col-md-4 event-item__owner'>
+          <div className="col-md-4 event-item__owner">
             <UserCard owner={props.owner} />
           </div>
         </div>
```
Comment (yushevchuk) [inline client/src/events/components/EventItem.js]: Remove space, please.

Comment (chooidos) [reply client/src/events/components/EventItem.js]: done

File: client/src/events/components/EditEventForm.js (hunk 1 above)
Comment (chooidos) [reply client/src/events/components/EditEventForm.js]: done

**Pull request merged**